src/BugsnagUnity/Native/Cocoa/NativeDeviceWithState.cs

```diff
@@ -22,6 +22,6 @@ public NativeDeviceWithState(IntPtr nativeDevice) : base(nativeDevice)
 
         public string Orientation { get => NativeWrapper.GetNativeString(ORIENTATION_KEY); set => NativeWrapper.SetNativeString(ORIENTATION_KEY,value); }
 
-        public DateTime? Time { get => DateTime.Parse(NativeWrapper.GetNativeString(TIME_KEY)); set => NativeWrapper.SetNativeString(TIME_KEY, value?.ToLongTimeString()); }
+        public DateTimeOffset? Time { get => DateTimeOffset.Parse(NativeWrapper.GetNativeString(TIME_KEY)); set => NativeWrapper.SetNativeString(TIME_KEY, value?.ToString()); }
```

Comment:
Is `ToString()` here guaranteed to give us the same format regardless of the user's local culture?  I think using ISO format (`"o"`) would be safer?

Reply:
Done 👍